dd-sdk-android-gradle-plugin/src/test/kotlin/com/datadog/gradle/plugin/DdExtensionForgeryFactory.kt

```diff
@@ -18,6 +18,12 @@ internal class DdExtensionForgeryFactory : ForgeryFactory<DdExtension> {
             site = forge.aValueFrom(DatadogSite::class.java).name
             variants = mock()
             checkProjectDependencies = forge.aValueFrom(SdkCheckLevel::class.java)
+            remoteRepositoryUrl = forge.aStringMatching(
+                "https://[a-z]{4,10}\\.[com|org]/[a-z]{4,10}/[a-z]{4,10}\\.git"
+            )
+            mappingFilePath = forge.aStringMatching(
+                "([a-z]+)/([a-z]+)/([a-z]+)/mapping.txt"
+            )
```

Comment:
```suggestion
            mappingFilePath = forge.aNullable {
                aStringMatching("([a-z]+)/([a-z]+)/([a-z]+)/mapping.txt")
            }
```

Reply:
Yes I know but I wanted to follow the pattern as we are using non - nullable generators for all the other properties in the extension (even though they are nullable). If we want to to use this here then it will make sense to change it for all the others and adapt the tests. I did not think this should be the topic for this PR.